From 53c8c71396b6e29b665996bc90d2a1853693466b Mon Sep 17 00:00:00 2001
From: Chris Morin <cmtm@google.com>
Date: Tue, 4 Dec 2018 16:45:59 -0800
Subject: [PATCH] don't listen on audit socket

Audit messages are already forwarded to the kernel message buffer since
we don't have auditd running. Listening on the audit socket would result
in duplicate messages showing up in the logs.
---
 src/journal/journald-server.c | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/src/journal/journald-server.c b/src/journal/journald-server.c
index 4850252897..a2a2e36f12 100644
--- a/src/journal/journald-server.c
+++ b/src/journal/journald-server.c
@@ -1965,12 +1965,8 @@ int server_init(Server *s) {
         if (r < 0)
                 return r;
 
-        /* Unless we got *some* sockets and not audit, open audit socket */
-        if (s->audit_fd >= 0 || no_sockets) {
-                r = server_open_audit(s);
-                if (r < 0)
-                        return r;
-        }
+        /* Don't listen on audit socket */
+        (void)no_sockets;
 
         r = server_open_kernel_seqnum(s);
         if (r < 0)
-- 
2.21.0.1020.gf2820cf01a-goog

